text/0002-searchable-display-attributes-behaviors.md

```diff
@@ -0,0 +1,123 @@
+- > **is not in `searchable_attributes` and not in `displayed_attributes`:**
+  > meilisearch will **not** use the field for a search and the field will **not** be returned in search results. The field is unused but should be kept in case of it is added in `searchable_attributes` or `displayed_attributes`.
+
+#### add a new configuration `attributes_order`?
+
+This new config may define the order of attributes for the criterion `Attributes`, no wildcard allowed.
+if `searchable_attribute` is: `["title", "desc"]`
+and `attributes_order` is: `["desc", "id", "title"]`
+The searchable attributes order would be:
+`"desc" > "title"` (`"id"` is not searchable so is ignored by the criterion `Attributes`).
+
+`attributes_order` is always exhaustive, If a new field is added, it is "pushed" at the end of the list.
+
+##### drawbacks:
+- new config added
+- more complexity
+- could be replace by a small note about wildcard in `searchable_attributes`
+
+#### Related issues
+
+- https://github.com/meilisearch/MeiliSearch/issues/1066
+- https://github.com/meilisearch/MeiliSearch/issues/942
+
+### Changes in meilisearch documentation
+
+Depending on what we choose, we'll have to add **default** and **wildcard** behavior of configurations, we'll potentially add a new configuration `attributes_order`. 
+
+<!-- Explain the proposal as if it was already included in MeiliSearch and you were teaching it to another user. That generally means:
+
+- Introducing new named concepts.
+- Explaining the feature largely in terms of examples.
+- The API for this feature, HTTP, CLI or config.
+- Explaining how the user should think about the feature and how it should impact the way they use MeiliSearch. It should explain the impact as concretely as possible.
+- If applicable, provide sample error messages, deprecation warnings, or migration guidance.
+
+If the changes modify the HTTP API, provide a description of the method, URL, parameters, body, status code, errors, etc...
+
+If it modifies the CLI, provide the env variable name, the argument name, and the description.
+
+This serves as a user-level guide. Anything that the user may encounter during its interaction with the feature should be presented here.
+Impact on documentation
+
+If the feature requires additions to the documentation or if sections of the documentation need to be updated because of this feature, it should be mentioned here. It's the role of the documentation team to point out the sections of the documentation that need to be updated. -->
+
+## Second Section: Technical Specifications
+
+<!-- This section has a much narrower audience: the developer that will implement the feature. Its goal is to make it as clear as possible to develop the feature, share knowledge, and think about the possibilities. -->
+
+### Architecture
+
+This change will impact **settings**, the **indexer** and the **search-engine** (`Attribute` criterion).
```

Comment:
```suggestion
This change will impact **settings**, the **indexer** and the **search-engine** (`Attributes` criterion).
```

Reply:
Should be lowercase if put between backticks